lib/Display/display.cpp

```diff
@@ -254,17 +260,100 @@ void display::displayHighscore()
     tft.print("reset highscores");
 }
 
+void display::displayHighscoreDecorative()
+{
+    displayCharacter(200, 100);
+
+    // items left
+    tft.fillRect(0, 0, 40, 40, COLOR_GOLD);                     //5
+    tft.drawRect(0, 0, 40, 40, ILI9341_ORANGE);
+    tft.drawRect(1, 1, 38, 38, ILI9341_ORANGE);
```

Comment:
make these into functions either separate functions for each type of block or preferably one function to make all of them